### PR TITLE
E401 & F841

### PR DESCRIPTION
**Pull request by guedou**
Small PR for #1277.

### CODE REVIEW

File: scapy/layers/tls/cert.py [hunk 1]
```diff
@@ -827,10 +827,6 @@ def import_from_asn1pkt(self, crl):
                 date = cert.revocationDate.val
                 if date[-1] == "Z":
                     date = date[:-1]
-                try:
-                    revocationDate = time.strptime(date, "%y%m%d%H%M%S")
```
Comment (gpotter2) [inline scapy/layers/tls/cert.py]: I think this was supposed to test if the date had a correct format or not

Comment (guedou) [reply scapy/layers/tls/cert.py]: I will flag it with `# noqa: F841`.

Comment (gpotter2) [reply scapy/layers/tls/cert.py]: I think removing `revocationDate =` should do the trick

File: scapy/contrib/bp.py [hunk 2]
```diff
@@ -72,7 +72,6 @@ def mysummary(self):
             tmp += ' S'
         if (self.ProcFlags & 0x20):
             tmp += ' ACKME'
-        RAWCOS = (self.ProcFlags & 0x0180)
         COS = (self.ProcFlags & 0x180) >> 7
```
[the remark this answers is not in the document]
Comment (guedou) [reply scapy/contrib/bp.py]: c

File: scapy/layers/inet.py [hunk 3]
```diff
@@ -1332,7 +1337,6 @@ def world_trace(self):
                 trt[trace_id] = loctrace
 
         # Load the map renderer
-        fig = plt.figure(num='Scapy')
```
[the remark this answers is not in the document]
Comment (guedou) [reply scapy/layers/inet.py]: fixed

**Pull request merged**